concepticondata/conceptlists/Desrochers-2010-330.tsv

```diff
@@ -299,7 +299,7 @@ Desrochers-2010-330-297	297	ofuscar	obfuscate			Verb	3.02941176470588	7
 Desrochers-2010-330-298	298	ornar	decorate	DECORATE	1739	Verb	2	5	7	
 Desrochers-2010-330-299	299	pagar	pay	PAY	718	Verb	6.84	5	342	6.75
 Desrochers-2010-330-300	300	parar	stop	STOP DOING	3351	Verb	6.47058823529412	5	200	6.64
-Desrochers-2010-330-301	301	partir	share			Verb	5.88235294117647	6	79	5.78
+Desrochers-2010-330-301	301	partir	share	SHARE	884	Verb	5.88235294117647	6	79	5.78
```

Comment:
'partir' is either `BREAK (CLEAVE)` or `LEAVE`. 'compartir' would be `SHARE`.

Reply:
I agree.